src/main/client/batch_write.c

```diff
@@ -487,11 +505,17 @@ static PyObject *AerospikeClient_BatchWriteInvoke(AerospikeClient *self,
                                        Py_None);
             }
         }
+        // else {
+        //     printf("aerospike_batch_write record:%d failed:%d", i, *result_code );
+        // }
     }
 
     goto CLEANUP3;
 
 CLEANUP0:
+    if (py_meta) {
```

Comment:
Could we take out the if statement since Py_XDECREF checks if the PyObject is null?